fixtures/flight/server/cli.server.js

```diff
@@ -6,9 +6,12 @@ register();
 const babelRegister = require('@babel/register');
 
 babelRegister({
+  babelrc: false,
   ignore: [/\/(build|node_modules)\//],
```

Comment:
Let's add `./server/` to the ignore list instead so we can use the preset for CJS.